src/smooth move animations.js

```diff
@@ -0,0 +1,109 @@
+/**
+👣
+@file smooth move animations
+@summary Define unique walking and idle tiles in each direction for any event.
+@license MIT
+@author Violgamba (Jon Heard)
+
+
+@description
+Builds on the smooth-move plugin to let you define unique walking and idle tiles in each direction
+for any event.  Useful if using bipsi for a more graphically involved game.
+
+NOTE - This plugin requires the smooth-move plugin in order to work.  It also needs to come AFTER
+smooth-move, so set it's "plugin-order" to something higher than what the smooth-move plugin's
+plugin-order is set to.
```

Comment:
something to consider here: if this plugin requires `smooth move` and has some additional setup complexity bc of that, would it be better if it just included `smooth move` via es6 imports and the two were self-contained/mutually exclusive?

Reply:
I'm not so familiar with this.  Are you suggesting to bundle `smooth-move` _into_ `smooth-move-animations`?  Or are you suggesting to replace the "plugin-order" step by having one imported plugin directly reference another imported plugin via es6 imports?

In terms of bundling: I could certainly do that.  My concern is that I think it might be more complex for the user to understand: one plugin does one thing, the other does the same thing, but also another, extra thing.

Reply:
yeah, i'm suggesting they be bundled together

you're right that them including overlapping features may be confusing, but on the other hand this plugin is useless without the other and requires extra manual steps in order to set them up in a compatible way. if they were bundled together, the setup would be simplified

Reply:
Hmm.  I'm inclined to leave them separate since it's conceptually more straightforward, but I'll think further about this.